pkg/sql/logictest/testdata/logic_test/create_index

```diff
@@ -403,3 +403,18 @@ CREATE INDEX ON opclasses(b blah_ops)
 # inverted, even if the type is invertible.
 statement error operator class "blah_ops" does not exist
 CREATE INDEX ON opclasses(c blah_ops)
+
+# This subtest ensures that creating a hash-sharded index results
```

Comment:
I think we have tests coverage for it, see it in `hash_sharded_index` logic test, it's fine to remove this. Looks like the problem is that `create_index` is not turned on in declarative schema changer so we never caught it there. Thanks for fixing this though!

Reply:
yup, you're right. I'll remove it.